tox.ini

```diff
@@ -0,0 +1,25 @@
+[tox]
+envlist = py{27,35}-django{18,19}, quality
+
+[testenv]
+setenv =
+    PYTHONPATH = {toxinidir}
+
+deps =
+    django18: Django>=1.8,<1.9
+    django19: Django>=1.9,<1.10
+    -rtest_requirements.txt
+
+commands =
+    django-admin.py test edx_rest_framework_extensions --settings=test_settings --with-coverage --cover-package=edx_rest_framework_extensions
+    coverage report
+
+[testenv:quality]
+deps =
+    django18: Django>=1.8,<1.9
+    django19: Django>=1.9,<1.10
```

Comment:
Could you set these django dependencies up in the base `[tox]` config so that they apply to all environments?